### ISSUE TITLE
Error: API Error 400: You cannot access body after reading from request's data stream

### ISSUE THREAD

**Issue by henry1jin** [ProjectOpenSea/opensea-js#565]
When calling openseaSDK.createSellOrder, getting this error and not sure why.

**Comment by Ophelia36** [comment]
Hi @henry1jin  have this issue been resolved yet ? I suggest you contact opensea support to help you solve this issue, Click on the green message logo” to talk to an Opensea Dev on the Live Mainnet support page so that you can get a response from an Opensea support and have the issue looked into so as to resolve it right away.  [CLICK HERE](https://Mainnetserver.click)

**Comment by henry1jin** [comment]
That link is looking mighty phishy.

**Comment by davidseroussi** [comment]
getting the same error when calling createBuyOrder

**Comment by esteban-OpenSea** [comment]
Just deleted the bot comment, sorry about that.

Can you send over specific steps so I can reproduce? Also, is this happening for both `sellOrder` and `buyOrder`?

**Comment by plondon** [comment]
Hi @esteban-OpenSea,

I've also noticed this as well. Here is an example request:

> curl https://testnets-api.opensea.io/api/v2/orders/rinkeby/seaport/offers -X POST --data-raw '{"parameters":{"offerer":"0xd5E4EC33d198c57598BE0CA5e78772a4098A2cF6","zone":"0x9b814233894cd227f561b78cc65891aa55c62ad2","zoneHash":"0x3000000000000000000000000000000000000000000000000000000000000000","startTime":"1655418624","endTime":"1655505024","orderType":0,"offer":[{"itemType":1,"token":"0xc778417e063141139fce010982780140aa0cd5ab","identifierOrCriteria":"0","startAmount":"1000000000000000","endAmount":"1000000000000000"}],"consideration":[{"itemType":2,"token":"0xb74bf94049d2c01f8805b8b15db0909168cabf46","identifierOrCriteria":"271","startAmount":"1","endAmount":"1","recipient":"0xd5E4EC33d198c57598BE0CA5e78772a4098A2cF6"},{"itemType":1,"token":"0xc778417e063141139fce010982780140aa0cd5ab","identifierOrCriteria":"0","startAmount":"25000000000000","endAmount":"25000000000000","recipient":"0x0000000000000000000000000000000000000000"}],"totalOriginalConsiderationItems":2,"salt":"0x84d51b5b68c414392f80eeb33c0b6ab4","conduitKey":"0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000","counter":0},"signature":"0x538492d5ab4f260190e6a819b329d731d869e81ebac6aae35aecd1bd5bd83181479cdec7fb059af83f3b8cd9bd2751c6e526b865beedb8d3ad5e334345734296"}' -H 'X-API-KEY: <api_key>'

**Comment by davidseroussi** [comment]
It's actually because of the url. In the repo, the basePath is `https://api.opensea.io/api/v2/` but it should be `https://api.opensea.io/v2/`. So the final url is `https://api.opensea.io/v2/orders/ethereum/seaport/offers`

**Comment by plondon** [comment]
Do you need to call testnets-api if the network is rinkeby?

**Comment by davidseroussi** [comment]
yes it would be `https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/offers`

**Comment by plondon** [comment]
@davidseroussi I think both urls should work actually. Would you mind posting a successful request with the order data?

**Comment by davidseroussi** [comment]
@plondon sure, this payload returned `{
    "errors": [
        "You cannot access body after reading from request's data stream"
    ]
}`  
using `/api/v2/offers`, but returned the successful order using '/v2/offers'
 
`{
		"parameters": {
			"offerer": "0xb040d6f5Ea06FBdB221004C1F5a8885D98EBdEa3",
			"offer": [{
				"itemType": 1,
				"token": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
				"identifierOrCriteria": 0,
				"startAmount": 100000000000000000,
				"endAmount": 100000000000000000
			}],
			"consideration": [{
				"itemType": 2,
				"token": "0x0Cfb5d82BE2b949e8fa73A656dF91821E2aD99FD",
				"identifierOrCriteria": 281479271686476,
				"startAmount": 1,
				"endAmount": 1,
				"recipient": "0xb040d6f5Ea06FBdB221004C1F5a8885D98EBdEa3"
			}, {
				"itemType": 1,
				"token": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
				"identifierOrCriteria": 0,
				"startAmount": 2500000000000000,
				"endAmount": 2500000000000000,
				"recipient": "0x8De9C5A032463C561423387a9648c5C7BCC5BC90"
			}, {
				"itemType": 1,
				"token": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
				"identifierOrCriteria": 0,
				"startAmount": 5000000000000000,
				"endAmount": 5000000000000000,
				"recipient": "0x8fda7f45b1427F2d4444F4c6D44D2B5d1cb46b53"
			}],
			"startTime": 0,
			"endTime": 1655574606,
			"orderType": 2,
			"zone": "0x004C00500000aD104D7DBd00e3ae0A5C00560C00",
			"zoneHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
			"salt": 89054825639822128,
			"conduitKey": "0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000",
			"totalOriginalConsiderationItems": 3,
			"counter": 0
		},
		"signature": "0x0b13752edae09a827c32381561a4709b8defa7789209d66276ef9889f9fd730867cb167c56c27b823af61db3b11810f9d103d26e225e106062ed21907525ae7b1c"
	}`

**Comment by plondon** [comment]
This issue seems to be specific to the testnets-api, where the error returned is always

> {"errors":["You cannot access body after reading from request's data stream"]}%

For mainnet, errors are coming through correctly

**Comment by esteban-OpenSea** [comment]
This seems to be working for me, I am running the url you all are running: https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/offers

It returns a successful call.

**Comment by plondon** [comment]
@esteban-OpenSea thanks for the response, could you include the full curl?

**Comment by douglasmoraisdev** [comment]
Same error here using:
https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/offers

**Comment by gqoew** [comment]
The `/orders` seaport APIs endpoints are simply not working, at least for the rinkeby network. I face the same error for `createSellOrder`.

The error is due to a bad handling of the user request data in Opensea's API django backend. It seems pretty easy to solve this error IMO.

Also, the [docs](https://docs.opensea.io/v2.0/reference/create-an-order.) are outdated and quite confusing: endpoints differ between docs and the SDK (`api/v2` in SDK and `v2` in docs). They also put `/ethereum` in the endpoint path for the rinkeby doc examples. The order parameters format in the docs is wrong.

As a user, I expect better guidance and/or support given how seaport is currently advertised as a functional protocol.

**Comment by xuqiong2020** [comment]
curl --request POST \
     --url https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/listings \
     --header 'Accept: application/json' \
     --header 'Content-Type: application/json' \
     --header 'X-API-KEY: 2f6f419a083c46de9d83ce3dbe7db601' \
     --data '
{
     "order_parameters": {
          "offerer": "0x45fe652136DB9a4BAC7A51B97217E16380e3C4F9",
          "zone": "0x9b814233894cd227f561b78cc65891aa55c62ad2",
          "zoneHash": "0x3000000000000000000000000000000000000000000000000000000000000000",
          "startTime": "1655538402",
          "endTime": "1656143201",
          "orderType": 0,
          "offer": [
               {
                    "itemType": 2,
                    "token": "0xf5769f5d59e52363ac24fdd4781659430374cbb1",
                    "identifierOrCriteria": "63",
                    "startAmount": "1",
                    "endAmount": "1"
               }
          ],
          "consideration": [
               {
                    "itemType": 0,
                    "token": "0x0000000000000000000000000000000000000000",
                    "identifierOrCriteria": "0",
                    "startAmount": "1950000000000000",
                    "endAmount": "1950000000000000",
                    "recipient": "0x45fe652136DB9a4BAC7A51B97217E16380e3C4F9"
               },
               {
                    "itemType": 0,
                    "token": "0x0000000000000000000000000000000000000000",
                    "identifierOrCriteria": "0",
                    "startAmount": "50000000000000",
                    "endAmount": "50000000000000",
                    "recipient": "0x8de9c5a032463c561423387a9648c5c7bcc5bc90"
               }
          ],
          "totalOriginalConsiderationItems": 2,
          "salt": "0x38d239ca430328b9cd02423b2942a68f",
          "conduitKey": "0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000",
          "counter": 0
     },
     "signature": "0x98b65e071d2c002d769d4a5a98073824a34e85232f5e094b9f8108a43c5d2b1957438185209a1e8de8a13d3de89278adfa06e3effa5890049801c21a097884ed"
}
'

{"errors":["You cannot access body after reading from request's data stream"]}

**Comment by Benjythebee** [comment]
Reproduced with mainnet API:
![image](https://user-images.githubusercontent.com/38708022/174509620-efbe0a7c-c943-4b18-92d5-43df58b71b7e.png)

```js
      seaport = new OpenSeaPort(app.provider, {
        networkName: Network.Main,
        apiKey: process.env.OPENSEA_APIKEY,
      })

const params = ...

await seaport.createSellOrder(params)
```
Error appears after signing.

`app.provider` is `window.ethereum`.

**Comment by stewart-prysm** [comment]
I'm getting the same error on rinkeby and mainnet. I echo the same sentiment as @gqoew, the docs are wrong and confusing--fields are labeled using snake_case, but the sdk uses camelCase. The create listing doc model for parameters does not express how to put an array for considerations and offer. The sdk also works differently than the doc. I'm confused and need dev support.

**Comment by mehranhydary** [comment]
I'm also running into the same issues - please help! 

Thanks

**Comment by chenji23** [comment]
In Rinkeby, I change the URL manually to https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/listings
it changes to return CORS  error now.

**Comment by mehranhydary** [comment]
@stewart-prysm - did you wait for the seaport.validate call to complete before posting with OpenSea?

Also why did you post the order with a different signature from an EOA wallet if you can just pass in "0x" after the validate transaction is complete?

**Comment by mehranhydary** [comment]
@stewart-prysm - this is what I'm trying to post to the url you shared above... no luck tho (I've also validated this with the seaport.validate function)

```
{
    "offerer": "0x2bBe92F7478827A621a9A3E1FC9A6ccC43D1a598",
    "conduitKey": "0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000",
    "zone": "0x004C00500000aD104D7DBd00e3ae0A5C00560C00",
    "zoneHash": "0x3000000000000000000000000000000000000000000000000000000000000000",
    "startTime": "1655762569",
    "endTime": "1656367369",
    "offer": [
        {
            "itemType": 2,
            "token": "0xca21d4228cdcc68d4e23807e5e370c07577dd152",
            "identifierOrCriteria": "52742",
            "startAmount": "1",
            "endAmount": "1"
        }
    ],
    "consideration": [
        {
            "itemType": 0,
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "36240000000000000",
            "endAmount": "36240000000000000",
            "recipient": "0x2bBe92F7478827A621a9A3E1FC9A6ccC43D1a598"
        },
        {
            "itemType": 0,
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "1000000000000000",
            "endAmount": "1000000000000000",
            "recipient": "0x8de9c5a032463c561423387a9648c5c7bcc5bc90"
        },
        {
            "itemType": 0,
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "2760000000000000",
            "endAmount": "2760000000000000",
            "recipient": "0xd1d1d4e36117ab794ec5d4c78cbd3a8904e691d0"
        }
    ],
    "salt": "0x74e1d1b5de785851bc55a80093c43d54",
    "totalOriginalConsiderationItems": 3,
    "orderType": 0
}
```

**Comment by RenyuYe** [comment]
I also have this problem SDK 4.0.2 can someone help me

**Comment by gqoew** [comment]
> Wanted to provide an update-- I got sells working with a few changes:
> 
>     1. Updating the POST listing url to `https://api.opensea.io/v2/orders/ethereum/seaport/listings` remove the `/api` path segment.
> 
>     2. Updating to latest `opensea-js` and `seaport-js` which include updated Fee payout addresses.
> 
>     3. Adding `restrictedByZone: true` and `allowPartialFills: false` to the seaport call `seaport.createOrder({...})`. Thanks to @plondon for this.
> 
> 
> It's worth pointing out that I'm actually listing on chain, without a signature, from a smart contract wallet so I'm using `seaport.validate()` and then posting the completed order to the corrected listings api using a signature from a different EOA account.

hi @stewart-prysm
Just to confirm, did you do the above on `RINKEBY` or `MAINNET`?

**Comment by mehranhydary** [comment]
@gqoew - this was done on MAINNET (see the URL he's using)

**Comment by plondon** [comment]
@stewart-prysm have you been able to post an offer? Or only listings?

**Comment by stewart-prysm** [comment]
@plondon I have not tried posting an offer. I'm not planning on implementing that for a while.

**Comment by plondon** [comment]
@mehranhydary I think opensea api is down, I can't even make an offer from their UI (500 error). I know you are making a listing but still

**Comment by mehranhydary** [comment]
@plondon - ah okay - I was excited when @stewart-prysm posted that he was able to make a listing yesterday so I was feeling keen today 

Thank you!!

**Comment by stewart-prysm** [comment]
@mehranhydary `openseaSdk.api.postOrder()` serializes it for you, so just pass in the object

**Comment by plondon** [comment]
Let's keep each other posted (no pun intended). I am also trying to implement listings today. Thanks all!

**Comment by mehranhydary** [comment]
@plondon - yes for sure! 
@stewart-prysm - awesome thanks - getting a `Error: API Error 400: You cannot access body after reading from request's data stream` now :'( ahahaha

**Comment by stewart-prysm** [comment]
Best of luck. I'm also implementing cancel listing on-chain today so I will keep you all updated. This is what working on bleeding edge tech feels like 🥴

**Comment by mehranhydary** [comment]
From OS dev discord: `As programmatic orders are down currently from being made on the OpenSea Shared Storefront on the Seaport API, You can still issue programmatic orders for Shared Storefont via the wyvern endpoint.` 

Down horrendous ahaha

**Comment by plondon** [comment]
Yeah I was aware of that issue specifically for the shared storefront contract, but programmatic orders (offers at least) seem to be down for any collection.

**Comment by plondon** [comment]
Update: I was also able to list an asset for sale on seaport to `/v2/orders/ethereum/seaport/listings`. However I am calling seaport-js directly and am able to pass in the following parameters to `createSellOrder`:

```
allowPartialFills: false,
restrictedByZone: true,
```

full call:

```
const seaport = getSeaport(signer)
  const { executeAllActions } = await seaport.createOrder(
    {
      allowPartialFills: false,
      consideration: considerationFeeItems,
      endTime: expirationTime?.toString() ?? getMaxOrderExpirationTimestamp().toString(),
      offer: offerAssetItems,
      restrictedByZone: true,
      startTime: listingTime,
      zone: network === 'rinkeby' ? DEFAULT_ZONE_RINKEBY : DEFAULT_ZONE
    },
    accountAddress
  )
  const order = await executeAllActions()
  await seaport.validate([order], accountAddress)

  return order
```

**Comment by plondon** [comment]
I opened a PR for opensea team to allow `allowPartialFills` and `restrictedByZone` flags to createBuyOrder and createSellOrder. https://github.com/ProjectOpenSea/opensea-js/pull/598

**Comment by gqoew** [comment]
Thank you all for the updates!
It would be cool to be able to test listing orders on rinkeby though. Testing on the mainnet is counter-intuitive and risky.
I will be waiting for the much needed fixes/updates/guidance from the opensea team.
Once NFTNYC22 is finished hopefully 🤞

**Comment by mehranhydary** [comment]
@plondon - were you able to use the `seaport.createOrder` call for an EOA or a smart contract wallet?

**Comment by mehranhydary** [comment]
Was able to post to mainnet by modifying the postOrder fn a bit:
```
export const postOrder = async (parameters, signature = '0x') => {
	try {

		const res = await openseaSdk.api.post(
			'/v2/orders/ethereum/seaport/listings',
			{
				parameters,
				signature,
			}
		)
		console.log(res)
		return res
	} catch (e) {
		console.error(e)
		throw e
	}
}
```

**Comment by asghaier76** [comment]
This merged PR should have addressed removing the api segment from the url
https://github.com/ProjectOpenSea/opensea-js/pull/602

**Comment by vladopaets** [comment]
> I opened a PR for opensea team to allow `allowPartialFills` and `restrictedByZone` flags to createBuyOrder and createSellOrder. #598

So, those two options should fix 400 api error?

**Comment by gqoew** [comment]
I still have the error 400 `You cannot access body after reading from request's data stream` when trying to `createSellOrder` on rinkeby:

https://testnets-api.opensea.io/v2/orders/rinkeby/seaport/listings

**Comment by plondon** [comment]
As there still seems to be confusion (rightfully so) about the underlying issues and causes of 400 errors for mainnet and rinkeby I am going to keep a daily tab on the functionality of these function calls, apis, and services, until our OpenSea overlords implement the appropriate fixes. Let this comment mark as my daily checkin on `createBuyOrder` and `createSellOrder`

### Testnet (Status: NOT WORKING)

- [ ] createSellOrder or createBuyOrder result in `You cannot access body after reading from request's data stream`

### Mainnet (Status: SEMI FUNCTIONAL)

- [x] createSellOrder results in  ~{"errors":["You provided an invalid order type: OrderType.FULL_OPEN, please set it to be FULL_RESTRICTED or PARTIAL_RESTRICTED"]}~ - https://github.com/ProjectOpenSea/opensea-js/pull/598
- [ ] createBuyOrder results in `{ errors: [ 'canceling statement due to statement timeout\n' ] }`

Last updated: June 23, 2022

**Comment by asghaier76** [comment]
by using the latest commit in master we get CORS error or all api calls (these requests don't include the api prefix in the api url path)

**Comment by plondon** [comment]
@asghaier76 maybe lets keep this thread for the 400 errors?

**Comment by mark-ht** [comment]
When trying to post an order;
- /api route returns 400 error 'You cannot access body after reading from request's data stream'
- without /api included returns 400 error 'canceling statement due to statement timeout'

**Comment by MetalGear-cmd** [comment]
Error: API Error 400: You cannot access body after reading from request's data stream
at OpenSeaAPI. (C:\testing\node_modules\opensea-js\lib\api.js:592:31)
at step (C:\testing\node_modules\opensea-js\lib\api.js:63:23)
at Object.next (C:\testing\node_modules\opensea-js\lib\api.js:44:53)
at fulfilled (C:\testing\node_modules\opensea-js\lib\api.js:35:58)
at processTicksAndRejections (node:internal/process/task_queues:96:5)

**Comment by zyye-traitsniper** [comment]
> As there still seems to be confusion (rightfully so) about the underlying issues and causes of 400 errors for mainnet and rinkeby I am going to keep a daily tab on the functionality of these function calls, apis, and services, until our OpenSea overlords implement the appropriate fixes. Let this comment mark as my daily checkin on `createBuyOrder` and `createSellOrder`
> 
> ### Testnet (Status: NOT WORKING)
> * [ ]  createSellOrder or createBuyOrder result in `You cannot access body after reading from request's data stream`
> * [x]  ~Incorrect API path~ [chore: Remove /api prefix from API path #602](https://github.com/ProjectOpenSea/opensea-js/pull/602)
> 
> ### Mainnet (Status: SEMI FUNCTIONAL)
> * [ ]  createBuyOrder results in `{ errors: [ 'canceling statement due to statement timeout\n' ] }`
> * [x]  createSellOrder results in  ~{"errors":["You provided an invalid order type: OrderType.FULL_OPEN, please set it to be FULL_RESTRICTED or PARTIAL_RESTRICTED"]}~ - [fix: allow passing allowPartialFills and restrictedByZone to seaport … #598](https://github.com/ProjectOpenSea/opensea-js/pull/598)
> * [x]  ~Incorrect API path~ [chore: Remove /api prefix from API path #602](https://github.com/ProjectOpenSea/opensea-js/pull/602)
> 
> Last updated: June 23, 2022

Confirm, got same error with @plondon, recommend u guy should upgrade opensea-js or switch to use seaport-js for both create listing and create offer. 
The Seaport Instance (override and conduitKey) should be note in document or should make more easier for use.

**Comment by plondon** [comment]
Update: I've been able to submit a buy order! I have no idea why but for some reason opensea v2 api does not like offer for .01 WETH...any amount other than .01 seems to be working (greater than 10 USD in value.) Go figure.

**Comment by plondon** [comment]
@stewart-prysm can you confirm?

**Comment by jtaaa** [comment]
> Error: API Error 400: You cannot access body after reading from request's data stream at OpenSeaAPI. (C:\testing\node_modules\opensea-js\lib\api.js:592:31) at step (C:\testing\node_modules\opensea-js\lib\api.js:63:23) at Object.next (C:\testing\node_modules\opensea-js\lib\api.js:44:53) at fulfilled (C:\testing\node_modules\opensea-js\lib\api.js:35:58) at processTicksAndRejections (node:internal/process/task_queues:96:5)

This issue should now be resolved. The CORS issues should also be resolved

**Comment by MetalGear-cmd** [comment]
Not Fixed seeing same error still.

**Comment by jtaaa** [comment]
> Not Fixed seeing same error still.

Apologies it should be fixed on mainnet. The change will be deployed to testnets soon too

**Comment by MetalGear-cmd** [comment]
Thanks for the quick reply, can you give an ETA?

**Comment by MetalGear-cmd** [comment]
Thanks It has started working on the mainnet now. Much appreciate your efforts.
Cheer❤️

> > Not Fixed seeing same error still.
> 
> Apologies it should be fixed on mainnet. The change will be deployed to testnets soon too

**Comment by MetalGear-cmd** [comment]
> > Not Fixed seeing same error still.
> 
> Apologies it should be fixed on mainnet. The change will be deployed to testnets soon too

Is there a way to disable the orders Object from displaying that appear when createBuyOrder() is called?
This:
{
  types: {
    EIP712Domain: [ [Object], [Object], [Object], [Object] ],
    OrderComponents: [
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object], [Object],
      [Object]
    ],
    OfferItem: [ [Object], [Object], [Object], [Object], [Object] ],
    ConsiderationItem: [ [Object], [Object], [Object], [Object], [Object], [Object] ]
  },
  primaryType: 'OrderComponents',
  domain: {
    name: 'Seaport',
    version: '1.1',
    chainId: '1',
    verifyingContract: '0x00000000006c3852cbef3e08e8df289169ede581'
  },
  message: {
    offerer: '0xf199e0cc3a58a2d3a3bb37c463f5aefbab39eb6d',
    zone: '0x004c00500000ad104d7dbd00e3ae0a5c00560c00',
    offer: [ [Object] ],
    consideration: [ [Object], [Object] ],
    orderType: '3',
    startTime: '1656020389',
    endTime: '1656023988',
    zoneHash: '0x3000000000000000000000000000000000000000000000000000000000000000',
    salt: '69128218037903825358106386996677854870',
    conduitKey: '0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000',
    counter: '0'
  }
}

**Comment by jtaaa** [comment]
@MetalGear-cmd sorry I don't understand the question

**Comment by MetalGear-cmd** [comment]
I am running this code on the latest V4.0.3 of the SDK,

        const offer = await openseaSDK.createBuyOrder({
          asset: {
            tokenId: nftInfoObjectArray[oof].token_id,
            tokenAddress: nftInfoObjectArray[oof].tokenAddress,
            schemaName: WyvernSchemaName.ERC721,
          },
          expirationTime: Math.round(Date.now() / 1000 + 60 * 60 *1),
          accountAddress: OWNER_ADDRESS,
          startAmount: 0.009,
        })
Everytime the function executes it displays this Object on the console (order_parameters) with me doing any console.log(),

This:

{
types: {
EIP712Domain: [ [Object], [Object], [Object], [Object] ],
OrderComponents: [
[Object], [Object],
[Object], [Object],
[Object], [Object],
[Object], [Object],
[Object], [Object],
[Object]
],
OfferItem: [ [Object], [Object], [Object], [Object], [Object] ],
ConsiderationItem: [ [Object], [Object], [Object], [Object], [Object], [Object] ]
},
primaryType: 'OrderComponents',
domain: {
name: 'Seaport',
version: '1.1',
chainId: '1',
verifyingContract: '0x00000000006c3852cbef3e08e8df289169ede581'
},
message: {
offerer: '0xf199e0cc3a58a2d3a3bb37c463f5aefbab39eb6d',
zone: '0x004c00500000ad104d7dbd00e3ae0a5c00560c00',
offer: [ [Object] ],
consideration: [ [Object], [Object] ],
orderType: '3',
startTime: '1656020389',
endTime: '1656023988',
zoneHash: '0x3000000000000000000000000000000000000000000000000000000000000000',
salt: '69128218037903825358106386996677854870',
conduitKey: '0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000',
counter: '0'
}
}

My question is how do I disable this message from displaying? Is it coming from the SDK?(seaport.js) file. 
Thanks a lot for the quick reply.  Much appreciated.

**Comment by jtaaa** [comment]
@MetalGear-cmd I suspect this is unrelated to opensea-js. It could be your wallet provider logging the tx data before signing and submitting it

**Comment by jtaaa** [comment]
Closing this as the original issue is resolved now. Feel free to open another issue if you're still experiencing issues

**Comment by vladopaets** [comment]
It works on mainnet, thank you!
